Classes/Place.py

```diff
@@ -5,3 +5,69 @@ def __init__(self):
 		self.Description = ''
 		self.Name = ''
 		self.CanHaveBuildings = False
+	
+	def GetID(self):
+		return self.ID
+	
+	def SetID(self, Id):
+		self.ID = Id
+	
+	
+	def GetType(self):
+		return self.Type
+	
+	def GetType(self, Type):
+		self.Type = Type
+		
+	
+	def GetDescription(self):
+		return self.Description
+	
+	def GetDescription(self, Id):
+		self.ID = Id
+	
+	
+	def GetName(self):
+		return self.Name
+	
+	def GetName(self, name):
+		self.Name = name
+	
+	
+	def GetCanHaveBuildings(self):
+		return self.ID
+	
+	def GetCanHaveBuildings(self, canhavebuildings):
+		self.GetCanHaveBuildings = canhavebuildings
+
+
+
+
+
+class Town(Place):
+	def __init__(self):
+		self.CanGoTo = ''
+	
+	def GetCanGoTo(self):
+		return self.CanGoTo
+
+	def SetCanGoTo(self, cangoto):
+		self.CanGoTo = cangoto
+
```

Comment:
"CanGoTO" is really not a Town attrribute, but the parent's attribute.  And I think a better word/phrase would be appropriate.

Reply:
Changed